data/hai missions.txt

```diff
@@ -393,6 +397,7 @@ mission "Unfettered returning home"
 		has "Unfettered Tribute 3: done"
 		random < 40
 	on offer
+		log "Helped a juvenile Hai in Unfettered space seek refuge in Hai-home. Despite the ongoing hostilities, he seems he got a pretty warm welcome."
```

Comment:
Oops, I rewrote that last bit a few times and ended up with a fragment of different sentences, as Bladewood notes. And... yep, I've put too many things in on offer instead of on complete, apologies.